contrib/intel/jenkins/run.py

```diff
@@ -126,18 +126,21 @@ def intel_mpi_benchmark(core, hosts, mpi, mode, group, user_env, log_file, util)
         print(f"Skipping {mpi.upper} {imb.testname} as execute condition fails")
     print('-------------------------------------------------------------------')
 
-def mpich_test_suite(core, hosts, mpi, mode, user_env, log_file, util):
+def mpich_test_suite(core, hosts, mpi, mode, user_env, log_file, util, weekly=None):
 
     mpich_tests = tests.MpichTestSuite(jobname=jbname,buildno=bno,
                                        testname="MpichTestSuite",core_prov=core,
                                        fabric=fab, mpitype=mpi, hosts=hosts,
                                        ofi_build_mode=mode, user_env=user_env,
-                                       log_file=log_file, util_prov=util)
+                                       log_file=log_file, util_prov=util, weekly=weekly)
 
     print('-------------------------------------------------------------------')
     if (mpich_tests.execute_condn == True):
-        print(f"Running mpichtestsuite: Spawn Tests for {core}-{util}-{fab}-{mpi}")
-        mpich_tests.execute_cmd("spawn")
+        print(f"Running mpichtestsuitefor {core}-{util}-{fab}-{mpi}")
```

Comment:
can we rework these prints to do a build and then a run for {core}-{util}-{fab}-{mpi}